lib/v2/screens/login/login_screen.dart

```diff
@@ -1,26 +1,110 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:seeds/constants/app_colors.dart';
 import 'package:seeds/providers/services/navigation_service.dart';
 import 'package:seeds/v2/components/flat_button_long.dart';
+import 'package:seeds/design/app_theme.dart';
+import 'package:seeds/i18n/wallet.i18n.dart';
 
 /// Login SCREEN
 class LoginScreen extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
+    double height = MediaQuery.of(context).size.height;
     return Scaffold(
-        body: Column(
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: [
-            FlatButtonLong(
-              onPressed: () {},
-              title: "Claim invite code",
+        body: SingleChildScrollView(
+      child: Column(
+        children: [
+          Container(
+            height: height * 0.4,
+            decoration: const BoxDecoration(
+                image: DecorationImage(
+                    image: AssetImage(
+              "assets/images/login/background.png",
+            ))),
+          ),
+          Row(
```

Comment:
this entire row is already in the app in the splash screen. 

Its an SVG

Reply:
Fixed